### DISCUSSION TITLE
Nominatim update strategy

### DISCUSSION THREAD

**Discussion by kvalev** [photoprism/photoprism#1774] [Q&A] [answered]
Hi,

what is the update strategy of your private Nominatim instance? Do you use the auto-updater or you do it manually from time to time?

The reason why I am asking is that I noticed some incorrect OSM data I would like to correct and I am wondering when it will be propagated to the places API.

Thanks.

**Comment by lastzero** [comment, marked as answer]
The auto-updater for Nominatim is set up, but we also update manually from time to time. Photon, for example, is not updated automatically because the server version must be compatible with the data. On top of both is a layer for caching. It is currently manually cleared as needed. This will be gradually automated once we have gained experience with the interaction of the systems and know what the best caching strategy is.

**Reply by kvalev** [reply]
Thanks! So I guess I will ping you when the data has been corrected upstream, so that you can clear the cache (for the region if possible).

**Reply by lastzero** [reply]
Yes, that's possible. We can update and delete places based on country, city, and state. It will later also be possible to correct data that hasn't been updated in OSM yet.
